src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/DueDiligenceValidatorTest.java

```diff
@@ -291,6 +291,15 @@ void testErrorReportedWhenDiligenceFieldIsNull() {
         assertError(DueDiligenceDto.DILIGENCE_FIELD, validationMessage, errors);
     }
 
+    @Test
```

Comment:
can you also add test for no errors when it is at max length

Reply:
Sure thing

Reply:
Done 👍